conf/test.config

```diff
@@ -4,20 +4,32 @@
  * -------------------------------------------------
  * Defines bundled input files and everything required
  * to run a fast and simple test. Use as follows:
- *   nextflow run nf-core/proteomicslfq -profile test
+ *   nextflow run nf-core/proteomicslfq -profile test,<docker/singularity>
```

Comment:
Think this origins from the template, but [podman](https://podman.io/) is actually supported as well.